aws/resolve_customer/.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 1.1.4
```

Comment:
Start at 1.1.4?

Reply:
It was just there to illustrate how to use different versions. I added a commit to change the initial version to 0.0.1